### ISSUE TITLE
[Linux] [3.7.0] Spotube does not start anymore

### ISSUE THREAD

**Issue by brockhaus** [KRTirtho/spotube#1555]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

This might be related to Issue #1544, but this was a slightly different platform and a nightly build.
Now this is in the release version. I still can start (but not use) 3.6.0 but can not 3.7.0.

This is what I see on the console when starting spotube:

```
(process:41386): GLib-GObject-CRITICAL **: 12:30:45.199: g_object_new_is_valid_property: object class 'MyApplication' has no property named 'com.github.KRTirtho.Spotube'

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.376: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.376: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_handler_find: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): GLib-GObject-WARNING **: 12:30:45.377: invalid (NULL) pointer instance

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.377: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:41386): Gtk-CRITICAL **: 12:30:45.377: gtk_container_forall: assertion 'GTK_IS_CONTAINER (container)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.377: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:41386): GLib-GObject-CRITICAL **: 12:30:45.378: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:41386): CRITICAL **: 12:30:45.378: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

```

### Expected Behavior

spotube shout start again.. ;-)

### Steps to reproduce

1. I've got the "please install new version" dialog on my working 3.6.0 Client.
2. I installed the 3.7.0 ubuntu deb release.
3. Started in from command line with `spotube`.
4. spotube did not start and showed to log written above.
5. Deinstalled 3.7.0 and reinstalled 3.6.0
6. Now the client startet again (but still refuses to work by blocking with the "Please download.." dialog).

### Operating System

Ubuntu 22.04.4 LTS on 64bit PC, Gnome Version: 42.9, Windowmanager: Wayland.

### Spotube version

3.7.0

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by LunasShade** [comment]
I have the same issue on Arch.

**Comment by SrIzan10** [comment]
Same issue over on Flatpak and Arch

**Comment by SrIzan10** [comment]
By the way @brockhaus I'm sure you haven't tried clicking outside the update prompt. That works and can bring the app to a usable state for now.

**Comment by brockhaus** [comment]
@SrIzan10 Oh wow, indeed! Didn't tried that. This helps me until v.3.7.1 fixes this problem, I guess. Thanks! :)

**Comment by BlueTree242** [comment]
I can confirm i have this issue too. I was very excited to try the update :(

**Comment by udorronsoro** [comment]
Confirmed on mine too. Back to 3.6.0.

**Comment by Suero152** [comment]
Also happens to me. 6.6.32-1-MANJARO (64-bit)

**Comment by KRTirtho** [comment]
Seems to be caused by the window_manager plugin https://github.com/leanflutter/window_manager/issues/460

**Comment by KRTirtho** [comment]
Fixed on 8fc44ed6550e8b2b804991ff82df08afb1c94ca8

**Comment by SrIzan10** [comment]
when will a new version with this fix be released?

**Comment by hamkg** [comment]
Fixed already, thank you, Kingkor. I just downloaded the latest version 3.7.0+284 from nightly builds. As per usual, I picked the .rpm file and repackaged it to .txz format. Everything seems to run OK. Thanks for the quick solution, most appreciated!

**Comment by KRTirtho** [comment]
I'll try to release it as soon as possible. This was an unexpected bug that didn't show up till last minute. The local env kind of fooled me. Sorry for the inconvinience.

**Comment by KRTirtho** [comment]
For now use the Nightly build. It contains the fix which will soon be available via 3.7.1

**Comment by BlueTree242** [comment]
Looks like the flatpak has to be renamed to `oss.krtirtho.spotube` otherwise you get this: `** (spotube:2): WARNING **: 19:27:22.419: Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown` and app doesn't launch.

**Comment by youssefadly237** [comment]
same issue
OS: Linux Mint 21.3 x86_64
Kernel: 5.15.0-107-generic

**Comment by s-b-repo** [comment]
thx for all the hard work

> For now use the Nightly build. It contains the fix which will soon be available via 3.7.1

**Comment by Peperehobbits01** [comment]
> For now use the Nightly build. It contains the fix which will soon be available via 3.7.1

Except for flatpak users, we are stuck on the flatpak because most of us didn't have the choice, but here is no nightly build for us...

**Comment by BlueTree242** [comment]
> > For now use the Nightly build. It contains the fix which will soon be available via 3.7.1
> 
> Except for flatpak users, we are stuck on the flatpak because most of us didn't have the choice, but here is no nightly build for us...

You can build it yourself but it won't launch until you change the name of flatpak, it has to match this https://github.com/KRTirtho/spotube/blob/dev/linux/CMakeLists.txt#L5

**Comment by hamkg** [comment]
> > For now use the Nightly build. It contains the fix which will soon be available via 3.7.1
> 
> Except for flatpak users, we are stuck on the flatpak because most of us didn't have the choice, but here is no nightly build for us...

Are the new features in 3.7.0 that indispensable for you? If not, why not use the previous flatpak version (3.6.0). I'm sure that Kingkor will offer a 3.7.2 flatpak as soon as he can.
Remember that spotube is free software, available only thanks to a dedicated developer.

**Comment by KRTirtho** [comment]
Version 3.7.1 has been released. Flatpak build will also available in a few hours

**Comment by Peperehobbits01** [comment]
Oh great!

**Comment by BlueTree242** [comment]
> Version 3.7.1 has been released. Flatpak build will also available in a few hours

Hey flatpak does not work because of the issue i mentioned previously :(

**Comment by Peperehobbits01** [comment]
> > > For now use the Nightly build. It contains the fix which will soon be available via 3.7.1
> > 
> > 
> > Except for flatpak users, we are stuck on the flatpak because most of us didn't have the choice, but here is no nightly build for us...
> 
> Are the new features in 3.7.0 that indispensable for you? If not, why not use the previous flatpak version (3.6.0). I'm sure that Kingkor will offer a 3.7.1 flatpak as soon as he can. Remember that spotube is free software, available only thanks to a dedicated developer.

I am aware, I am a dev that makes open source software to. I have not said this whit any bad mining. Anyway, I just think a nigthly branch for flatpak would make sense, so you can test on all platform. Let's move on, the bugs is fixed now anyway because of is great work :)

**Comment by brockhaus** [comment]
> Version 3.7.1 has been released. Flatpak build will also available in a few hours

Works for me again. \o/
Thank you all!
